heutagogy/persistence.py

```diff
@@ -26,14 +26,16 @@ class Bookmark(db.Model):
     timestamp = db.Column(db.DateTime, nullable=False)
     url = db.Column(db.String, nullable=False)
     title = db.Column(db.String, nullable=False)
+    notes = db.Column(db.String, nullable=True)
```

Comment:
I would like to have it as a string rather than array of the string to simplify implementation on the frontend.